SPRINTLOG.md

```diff
@@ -409,3 +409,4 @@ _Nothing merged in CLI during this sprint_
 
 - Move raw Technical Overview doc to repo, add page numbers ([#1539](https://github.com/ScilifelabDataCentre/dds_web/pull/1539))
 - Small updates to Technical Overview contents ([#1540](https://github.com/ScilifelabDataCentre/dds_web/pull/1540))
+- Fixed index out of range when listing files from root ([#1542](https://github.com/ScilifelabDataCentre/dds_web/pull/1542))
```

Comment:
It has the newline, otherwise the prettier action would have failed.

Also, when a newlins is missing, GitHub flags the file with a red symbol marker. Pretty handy!
https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b